### ISSUE TITLE
fanscribed.com is down, and domain is lost

### ISSUE THREAD

**Issue by cweiske** [fanscribed/fanscribed#99]
The fanscribed.com domain is for sale now :/

**Comment by matthewryanscott** [comment]
Yes, I'm aware of that. :-/ The site had been offline for many months now, but I chose to let the domain lapse as the project never gained enough traction for me to afford spending time+money maintaining the domain. I've removed the link to fanscribed.com from the github description, as well as prefixing the description with an `[UNMAINTAINED]` indicator.

**Comment by cweiske** [comment]
Do you have a backup of the database? The "Logbuch Netzpolitik" podcast had a nearly finished transcription that wasn't public because it had not been reviewed. I'd be sad if that was lost.

**Comment by matthewryanscott** [comment]
Yes, I still have the database. Would you be able to get some usefulness out of a direct database dump (minus user accounts and password hashes of course)? If so I can prepare that post that as an attachment to the project.

If not, I would have to take some extra time to do this, but would be willing to put together an HTML export that has all of the transcribed text (regardless of whether reviewed or not).

**Comment by cweiske** [comment]
I can surely dig into the database schema, that should not be a problem.

**Comment by cweiske** [comment]
Could you send me the database dump?
